tools/docker-network/docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
       --valueLayer.snapshot.file=/tmp/assets/7R1itJx5hVuo9w9hjg5cwKFmek4HMSoBDgJZN8hKGxih.bin
       --syncbeacon.broadcastInterval=5
       --syncbeacon.startSynced=true
+      --metrics.manaResearch=true
```

Comment:
Shouldn't this be `false` by default?

Reply:
It is `false` by default, and only enabled for the faucet node (peer_master) in the docker network, because this is the node grafana (and prometheus) queries for data.